pack/pom.xml

```diff
@@ -40,6 +40,7 @@
             </activation>
             <properties>
                 <sakai.skip.tests>false</sakai.skip.tests>
+                <sakai.skip.phase>test</sakai.skip.phase>
```

Comment:
And this?

Reply:
This comes from the original vue PR https://github.com/sakaiproject/sakai/pull/10484#discussion_r863940978